docs/examples.md

```diff
@@ -6,9 +12,9 @@ Here you can find a bunch of simple examples for using `fcct`, with some explana
 
 This example modifies the existing `core` user and sets its ssh key.
 
-```yaml fedora-coreos-config
```

Comment:
These are how the `test` script knows to test the configs.  If we remove these, we'll need to find another solution for the script.